### PR TITLE
My finished project, week 3, business site

### PR DESCRIPTION
**Pull request by matildasvensson85**

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi! Can you submit a link to your netlify? :)

**Comment by matildasvensson85** [comment]
> Hi! Can you submit a link to your netlify? :)

Here is the link: https://focused-swirles-cdfc67.netlify.app/

File: code/index.html [hunk 1]
```diff
@@ -2,29 +2,88 @@
...
+      <h1 class="hero-headline">Adventure trips</h1>
+    </header>
+
+    <!-- CONTENT-->  
+    <div class="content-container">
 
     <!-- Signup form -->
+    <section class="signup-form">
+
+      <h2 class="h2-headline">Log in here</h2>
```
Comment (HIPPIEKICK) [inline code/index.html:26]: Try to come up with as descriptive class names as you can. Maybe just headline is enough here? 😇

File: code/index.html [hunk 2]
```diff
@@ -2,29 +2,88 @@
...
+
+        <label class="radio-label"> 
+          <input type="radio">
+          Male
+        </label>
+
+        <label class="radio-label"> 
+          <input type="radio">
+          Other
+        </label>
```
Comment (HIPPIEKICK) [inline code/index.html:58]: You need to give the radio buttons the `name` attribute and give them the same value in order for them to work as a radio button group

File: code/script.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
 function toggle() {
-  this.classList.toggle("active")
+  this.classList.toggle("open")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+document.getElementById("question-1").onclick = toggle
+document.getElementById("question-2").onclick = toggle
```
Comment (HIPPIEKICK) [inline code/script.js:12]: Nice and clean!

File: code/script.js [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
+
+const answer1 = document.getElementById('answer-1');
+const answer2 = document.getElementById('answer-2');
```
Comment (HIPPIEKICK) [inline code/script.js:15]: These are not used

File: code/style.css [hunk 5]
```diff
@@ -2,6 +2,136 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+.body {
```
Comment (HIPPIEKICK) [inline code/style.css:5]: Remove the `.` to style the body element

File: code/style.css [hunk 6]
```diff
@@ -2,6 +2,136 @@
...
+
+.signup-form {
+  width: 35vw;
+}
+
+.accordion {
+  width: 35vw;
+  display: block;
+}
+}
```
Comment (HIPPIEKICK) [inline code/style.css:135]: Nice! Just remember the indentation

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Well done Matilda! My main comment is fixing the radio buttons so that only one is clickable at a time. There's a section about it in the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475), so check that out as well as the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap indentation and conventions.

**Comment by KarolCislowski** [comment]
Hello, it's your project. Do you remember me? I may be imperfect, but I believe you can still change that before you sit down to your final masterpiece!
image

**Comment by matildasvensson85** [comment]
I made the changes, the updated Netlify page can be found here: project-business-website.netlify.app

**Review by KarolCislowski — APPROVED** [review]
good job 👍🏻

**Pull request closed without merging**